### ISSUE TITLE
gunzip -f error when all seems all right

### ISSUE THREAD

**Issue by poquirion** [SIMEXP/niak#287]
```
Something went bad ... the job has FAILED !
The last error message occured was :
niak:read: . There was a problem unzipping the file. Please check that the command 'gunzip -f' works, or change this command using the variable GB_NIAK_UNZIP in the file NIAK_GB_VARS
File /usr/local/niak/commands/read_write/niak_read_vol.m at line 236
File /usr/local/niak/commands/read_write/niak_cp_fmri.m at line 79
File /usr/local/niak/bricks/misc/niak_brick_copy.m at line 147
File /usr/local/niak/bricks/fmri_preprocess/niak_brick_resample_vol.m at line 227
File /usr/local/niak/bricks/t1_processing/niak_brick_anat2func.m at line 325
File /usr/local/niak/extensions/psom/psom_run_job.m at line 185
File /usr/local/niak/extensions/psom/psom_run_job.m at line 106
File /usr/local/niak/extensions/psom/psom_worker.m at line 161
```
I check and was able to unzio the file manually with system('gunzip -f /path/to/file.nii.gz')

**Comment by poquirion** [comment]
I get that on the gsp sample, and it seems random

**Comment by pbellec** [comment]
My bet is that the tmp is full. On guillimin, you should not use `/tmp`. It's sometimes full and in any case it's small. It should be set to $TEMPDIR.

**Comment by poquirion** [comment]
I am using localscratch

**Comment by surchs** [comment]
Also reads similar to #16

**Comment by poquirion** [comment]
Its seems that it is indeed using `/tmp` instead of `/localscratch`. Damn it!

**Comment by pbellec** [comment]
So then it would relate to `psom_gb_vars_local` not being properly detected, and/or the `niak_gb_vars` not properly grabbing the info from psom.

**Comment by pbellec** [comment]
or your `psom_gb_vars_local` using `/tmp` is another possibility of course...

**Comment by poquirion** [comment]
I still got the error with /localscratch 

```
Source : /gs/project/gsf-624-aa/poq/redo_oct_20/result/anat/sub-0252/anat_sub-0252_nuc_stereolin.nii.gz
Target : /gs/project/gsf-624-aa/poq/redo_oct_20/result/intermediate/sub-0252/motion_correction/motion_target_sub-0252_1_1.nii.gz
Resampling the anatomical image in the functional space ...
Copying or converting file /gs/project/gsf-624-aa/poq/redo_oct_20/result/anat/sub-0252/anat_sub-0252_nuc_stereolin.nii.gz to /localscratch/76952046.gm-1r16-n04.guillimin.clumeq.ca/niak_tmp_anat2func_sub-0252_223138249_anat_init/source.mnc


Something went bad ... the job has FAILED !
The last error message occured was :
niak:read: . There was a problem unzipping the file. Please check that the command 'gunzip -f' works, or change this command using the variable GB_NIAK_UNZIP in the file NIAK_GB_VARS
File /usr/local/niak/commands/read_write/niak_read_vol.m at line 236
File /usr/local/niak/commands/read_write/niak_cp_fmri.m at line 79
File /usr/local/niak/bricks/misc/niak_brick_copy.m at line 147
File /usr/local/niak/bricks/fmri_preprocess/niak_brick_resample_vol.m at line 227
File /usr/local/niak/bricks/t1_processing/niak_brick_anat2func.m at line 325
File /usr/local/niak/extensions/psom/psom_run_job.m at line 185
File /usr/local/niak/extensions/psom/psom_run_job.m at line 106
File /usr/local/niak/extensions/psom/psom_worker.m at line 161


The output file or directory /gs/project/gsf-624-aa/poq/redo_oct_20/result/anat/sub-0252/transf_sub-0252_nativefunc_to_stereolin.xfm has not been generated!
The output file or directory /gs/project/gsf-624-aa/poq/redo_oct_20/result/anat/sub-0252/anat_sub-0252_nativefunc_hires.nii.gz has not been generated!
The output file or directory /gs/project/gsf-624-aa/poq/redo_oct_20/result/anat/sub-0252/anat_sub-0252_nativefunc_lowres.nii.gz has not been generated!


```

**Comment by poquirion** [comment]
fixed in 0.18.0
